frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "@material-ui/lab": "latest",
     "@types/d3-scale-chromatic": "^3.0.0",
     "@vgrid/sass-inline-svg": "^1.0.1",
+    "client-zip": "^2.2.2",
```

Comment:
`github:seve/html-to-image#seve/add-cloned-tag` do we still need to point to your fork?

Reply:
nice catch 💀